### PR TITLE
Update validators.ca.xlf

### PR DESCRIPTION
**Pull request by rogerguasch**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Update validators.ca.xlf for Catalan language.

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.ca.xlf [hunk 1]
```diff
@@ -301,11 +301,35 @@
...
+                <source>Error</source>
+                <target>Error</target>
             </trans-unit>
             <trans-unit id="83">
                 <source>This is not a valid UUID.</source>
                 <target>Aquest valor no és un UUID vàlid.</target>
             </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Aquest valor ha de ser múltiple de {{ compared_value }}</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.ca.xlf]: There is a missing dot at the end of this translation. Thanks.

**Review by javiereguiluz — APPROVED** [review]
Thanks Roger!

File: src/Symfony/Component/Validator/Resources/translations/validators.ca.xlf (hunk 1 above)
Comment (rogerguasch) [reply src/Symfony/Component/Validator/Resources/translations/validators.ca.xlf]: Fiexed

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @rogerguasch.

**Pull request merged**